scripts/bundle-package.ts

```diff
@@ -21,11 +21,19 @@ interface Options {
   watch?: boolean;
 }
 
+const makeExternalPredicate = (externals: string[]) => {
```

Comment:
This might matter for some packages so you don't end up bundling stuff like `lodash/pickBy`, because without a solution like this `'lodash'` is treated as an external but all internal submodules are just bundled